sdk/cdn/arm-cdn/src/operations/customDomains.ts

```diff
@@ -30,9 +34,7 @@ import {
   CustomDomainsDisableCustomHttpsResponse,
   CustomDomainsEnableCustomHttpsOptionalParams,
   CustomDomainsEnableCustomHttpsResponse,
-  CustomDomainsListByEndpointNextResponse,
-  Profile,
```

Comment:
I think we should revert the relevant changes in https://github.com/Azure/azure-sdk-for-js/pull/23887/commits/92838949af3cd2de273724d9507310b9c2e736cf?